public/css/style.css

```diff
@@ -0,0 +1,83 @@
+/* Great source: https://css-tricks.com/hexagons-and-beyond-flexible-responsive-grid-patterns-sans-media-queries/ */
+
+#board {
```

Comment:
in general, we don't use ids very much in CSS, mostly classes. Because there may be more than one board per page...